common/frame.go

```diff
@@ -537,7 +539,7 @@ func (f *Frame) waitForExecutionContext(world executionWorld) {
 	f.log.Debugf("Frame:waitForExecutionContext", "fid:%s furl:%q world:%s",
 		f.ID(), f.URL(), world)
 
-	t := time.NewTimer(50 * time.Millisecond)
+	t := time.NewTicker(50 * time.Millisecond)
```

Comment:
`NewTimer` only runs once, but it looks like we want to run this more than once, it's why it's changed to `NewTicker`.

Reply:
Nice catch! 🏈

Reply:
Opened a new PR for this fix: https://github.com/grafana/xk6-browser/pull/539